libs/camera_rig_edex/camera_rig_edex.cpp

```
@@ -207,6 +204,11 @@ const Isometry3T& CameraRigEdex::getExtrinsic(uint32_t index) const {
 }
 
 ErrorCode CameraRigEdex::getFrame(Sources& sources, Metas& metas, Sources& masks_sources, DepthSources& depth_sources) {
+  sources.assign(cameras_.size(), {});
+  metas.assign(cameras_.size(), {});
+  masks_sources.assign(cameras_.size(), {});
```

Cppcheck (2.21.0)

[style] The function 'sqrt_epsilon' is never used.

(unusedFunction)